docs/running-on-device.md

```diff
@@ -43,6 +43,8 @@ emulator-5554 offline   # Google emulator
 
 Seeing `device` in the right column means the device is connected. You must have **only one device connected** at a time.
 
+> If you see `unauthorized` in the list you will need to run `adb reverse tcp:8081 tcp:8081` and press allow USB debugging on the device.
```

Comment:
```suggestion
:::note
If you see `unauthorized` in the list you will need to run `adb reverse tcp:8081 tcp:8081` and press allow USB debugging on the device.
:::
```

Let's use the [Admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of the `blockquote` - we are in process of replacing note like those (they are legacy).